ThemeStyle.js

```diff
@@ -0,0 +1,19 @@
+export default {
+    fontColor: "#6b6b6b",
+    fontSize: "1em",
+    colors: [
+        '#8949D5',
```

Comment:
are these colors from the vsd-react-ui?

Reply:
No, these colors are given by the Design Team.